### ISSUE TITLE
np.ma.argsort uses default fill_value not largest representable value

### ISSUE THREAD

**Issue by josef-pkt** [numpy/numpy#2890]
example for bug with wrong sorting: http://projects.scipy.org/scipy/ticket/1805

np.ma.sort uses fill_value=np.ma.minimum_fill_value(aam) which uses the largest representable value

fix should be to use this also in argsort

**Comment by eric-wieser** [comment]
Is this resolved by #8678?

**Comment by eric-wieser** [comment]
Looks good to me:
```
In [28]: np.sort(v3)
Out[28]:
masked_array(data = [-inf -1e+20 -0.3 0.0 1.0 2.0 100.0 5e+40 inf -- --],
             mask = [False False False False False False False False False  True  True],
       fill_value = 1e+20)
```
